### PR TITLE
fix: make bounds independent of scene

### PR DESCRIPTION
**Pull request by ad1992**

### CODE REVIEW

File: packages/excalidraw/element/bounds.ts [hunk 1]
```diff
@@ -75,23 +75,12 @@ export class ElementBounds {
...
-    // FIXME remove when we get rid of all the id:Scene / element:Scene mapping
-    const shouldCache = !!scene;
-
-    if (shouldCache) {
-      ElementBounds.boundsCache.set(element, {
-        version: element.version,
-        bounds,
-      });
-    }
+    ElementBounds.boundsCache.set(element, {
```
Comment (ad1992) [inline packages/excalidraw/element/bounds.ts:80]: do we need to set and retrieve from cache?

Comment (dwelle) [reply packages/excalidraw/element/bounds.ts:79]: of course, otherwise getting bounds would be a major bottleneck during hot paths.

Comment (dwelle) [reply packages/excalidraw/element/bounds.ts:79]: as for whether removing the `shouldCache` check won't break anything, I'm not sure. Can't reproduce the bug that this was fixing, but there may have been some edge case I don't recall.

**Review by dwelle — APPROVED** [review]

File: packages/excalidraw/element/bounds.ts (hunk 1 above)
Comment (ad1992) [reply packages/excalidraw/element/bounds.ts:79]: Yes my question was whether we need the check shouldCache anymore, as per the logic its not needed anymore

**Pull request merged**